docs/quickstart_guide.md

```diff
@@ -87,12 +90,37 @@ SSD-equipped nodes are highly recommended. Full list of CloudLab nodes can be fo
     >
     > This script can print `Command failed` when creating the devmapper at the end. This can be safely ignored.
 
+    > **Note:**
+    >
+    > [eStargz](https://github.com/containerd/stargz-snapshotter/tree/cmd/v0.12.1) is a
+    > lazily-pullable image format developed to improve the performance of container boot-ups by
+    > making better usage of the layering structure of container images. The image format is 
+    > compatible to [OCI](https://github.com/opencontainers/image-spec/)/[Docker](https://github.com/moby/moby/blob/master/image/spec/v1.2.md) images, therefore it allows pushing images to 
+    > standard container registries.
+    > To enable runs with `stargz` images, setup kubelet by adding the `stock-only` and `use-stargz`
```

Comment:
Maybe you can put this note before the previous command, as I only read it after I already ran the set-up script without the stargz option. Not sure if it is just my issue.

Reply:
Moved it before the standard command. Thank you!